lib/query.rb

```diff
@@ -98,6 +98,14 @@ def generate_filter(json)
     filter_from_fields(json, query_fields)
   end
 
+  def query_string_query(json, value, field)
```

Comment:
Can you call this `generate_query_string_query`? That would be more in line with other methods.